### PR TITLE
fix: create `lti_platform_registration` table upon a fresh installation

### PR DESCRIPTION
**Pull request by wazelin**

### CODE REVIEW

**Review by augustas — APPROVED** [review]
Didn't test, yet looking good

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by pedromarceOAT — APPROVED** [review]

File: manifest.php [hunk 1]
```diff
@@ -24,10 +24,11 @@
 use oat\taoLti\controller\CookieUtils;
 use oat\taoLti\controller\Security;
 use oat\taoLti\models\classes\ServiceProvider\LtiServiceProvider;
+use oat\taoLti\scripts\install\CreateLti1p3RegistrationSnapshotSchema;
```
Comment (kilatib) [inline manifest.php:27]:
```suggestion
* Copyright (c) 2013-2022 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
```

File: scripts/install/CreateLti1p3RegistrationSnapshotSchema.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    {
+        parent::__invoke($params);
+    }
+
+    protected function provideUsage(): array
+    {
+        return [
+            'prefix' => 'h',
+            'longPrefix' => 'help',
+            'description' => 'Shows this message'
```
Comment (kilatib) [inline scripts/install/CreateLti1p3RegistrationSnapshotSchema.php:46]: could you please add a more meaningful description?

**Review by kilatib — APPROVED** [review]

File: scripts/install/CreateLti1p3RegistrationSnapshotSchema.php (hunk 2 above)
Comment (wazelin) [reply scripts/install/CreateLti1p3RegistrationSnapshotSchema.php:46]: It is meaningful, if you try running it ;)

```
php index.php '\oat\taoLti\scripts\install\CreateLti1p3RegistrationSnapshotSchema' --helpxdebug.client_host/xdebug.client_port) :-(
Create `lti_platform_registration` table.

  Optional Arguments:
    -p persistence, --persistence persistence (default: "default")
    -h help, --help help
      Shows this message
```

Comment (wazelin) [reply scripts/install/CreateLti1p3RegistrationSnapshotSchema.php:46]: manifests were probably initially semi-automatically generated, we don't update the headers there as a rule.

**Pull request merged**